README.md

```diff
@@ -42,6 +42,7 @@ This package should only be used in projects starting from scratch, since it ove
         'REALM_ADMIN_ROLE': '<REALM_ADMIN_ROLE>',
         'EXEMPT_URIS': [],  # URIS to be ignored by the package
         'GRAPHQL_ENDPOINT': 'graphql/'  # Default graphQL endpoint
+        'DECODE_TOKEN': True  # Required since version 1.1.1
```

Comment:
- 'DECODE_TOKEN': True # Optional: Whether to decode token or use Keycloak API to introspect it
- 'VERIFY_AUDIENCE': True # Optional: Whether the token's audience should be verified. Requires Keycloak setup
- 'USER_INFO_IN_TOKEN': True # Optional: If the user info is in the token (used if DECODE_TOKEN is true)
- TRACE_DEBUG_LOGS: False # Optional: Whether to show the traceback of debug logs